### PR TITLE
Sections structure change

### PR DESCRIPTION
**Pull request by richfab**
## Context
In preparation of the gbfs.org redesign.

## What's Changed
Reorganized the sections of gbfs.org to improve clarity.
- Created new 'Get started' section with the Implementation Guide
- Merged 'Specification', 'Data Quality' and 'Learn' into 'Documentation'
- Renamed 'Toolbox' to 'Tools'
- Renamed 'Participate' to 'Community'
- Removed feed contact email tracker from 'Data Quality' because it's not maintained
- Removed Glossary
- Removed the top banner which had the Implementation Guide announcement

Before | After
-- | --
image |  image

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -99,46 +99,39 @@ plugins:
...
+            FAQ: FAQ
+            Tools: Outils
+            Community: Communauté
         - locale: es
           name: Español
           build: true
           nav_translations:
             Home: Hogar
-            Specification: Especificación
+            Get started: Primeros pasos
```
Comment (fredericsimard) [inline mkdocs.yml:116]: Shouldn't this be "Getting started" ?

File: mkdocs.yml [hunk 2]
```diff
@@ -99,46 +99,39 @@ plugins:
...
+            Data Quality: Calidad de los datos
+            Data Policy: Política de datos
+            FAQ: FAQ
+            Tools: Herramientas
+            Community: Comunidad
 
 nav:
   - Home: index.md
-  - Specification:
+  - Get started: guide.md
```
Comment (fredericsimard) [inline mkdocs.yml:128]: Shouldn't this be "Getting started" ?

File: mkdocs.yml [hunk 3]
```diff
@@ -99,46 +99,39 @@ plugins:
           build: true
           nav_translations:
             Home: Accueil
-            Specification: Spécification
+            Get started: Premiers pas
```
Comment (fredericsimard) [inline mkdocs.yml:102]: Shouldn't this be "Getting started" ?

File: docs/en/index.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # GBFS: A Common Language for Shared Mobility
 
 <div class="landing-page">
-    <a class="button" href="specification">Specification</a><a class="button" href="data-quality">Data Quality</a><a class="button" href="toolbox">Toolbox</a><a class="button" href="learn">Learn</a><a class="button" href="participate">Participate</a></div>
+    <a class="button" href="guide">Get started</a><a class="button" href="specification">Documentation</a><a class="button" href="tools">Tools</a><a class="button" href="community">Community</a></div>
```
Comment (fredericsimard) [inline docs/en/index.md:4]: Shouldn't this be "Getting started" ?

**Review by fredericsimard — COMMENTED** [review]
3 suggestions in the nav file and /en/index.md...

File: mkdocs.yml (hunk 1 above)
Comment (richfab) [reply mkdocs.yml:116]: Both are valid :) I took inspiration from https://www.swift.org/

**Review by fredericsimard — APPROVED** [review]
LGTM!

**Comment by richfab** [comment]
@fredericsimard Could you please create these redirects?

URL | Redirect to
-- | --
https://gbfs.org/learn/guide | https://gbfs.org/guide/
https://gbfs.org/fr/learn/guide | https://gbfs.org/fr/guide/
https://gbfs.org/es/learn/guide | https://gbfs.org/es/guide/

I tried to do it with the the redirects in [mkdocs.yml](https://github.com/MobilityData/gbfs.org/blob/master/mkdocs.yml#L80) but they don't seem to be working:
- I tried: `'learn/guide.md': 'guide.md'` but I get the warning "Redirect target 'guide.md' does not exist!"
- The only redirect that works is to an external URL.

Thanks in advance 🙏

**Comment by fredericsimard** [comment]
Yeah, from experience it doesn't work well. What I usually do is use a `.htaccess` file installed at the root of the website and that works really well. I have to modify the script that updates the website to move the file in the correct position after the update, but it's not a big issue.

All we'll have to do in the future is add redirections in this file.

**Comment by fredericsimard** [comment]
Turns out the whole thing was already in place, added the three new redirections: #103

**Comment by fredericsimard** [comment]
Website is updated and all redirects are working as expected.

**Pull request merged**